libs/components/src/search/search.component.ts

```diff
@@ -43,20 +47,34 @@ export class SearchComponent implements ControlValueAccessor, FocusableElement {
   // Use `type="text"` for Safari to improve rendering performance
   protected inputType = isBrowserSafariApi() ? ("text" as const) : ("search" as const);
 
-  readonly disabled = model<boolean>();
-  readonly placeholder = input<string>();
-  readonly autocomplete = input<string>();
+  protected isInputFocused = signal(false);
+  protected isFormHovered = signal(false);
+
+  protected showResetButton = computed(() => this.isInputFocused() || this.isFormHovered());
+
+  @Input() disabled: boolean;
+  @Input() placeholder: string;
+  @Input() autocomplete: string;
```

Comment:
Sorry about the merge conflicts, we just did a big migration to signal inputs for CL components. Can you put these back to how they are in `main`:
```suggestion
  readonly disabled = model<boolean>();
  readonly placeholder = input<string>();
  readonly autocomplete = input<string>();
```
And update their call sites back to how they are in `main`? 
- Line 97 in this file would go back to `this.disabled.set(isDisabled);`
- Line 20 in the template would call `placeholder()`
- Line 27 in the template would call `disabled()`
- Line 28 in the template would call `autocomplete()`

Reply:
That makes sense. Thanks for pointing this out. Updated.